docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  wordpress:
+    image: wordpress:latest
```

Comment:
should not be latest, instead rather use the latest version tag, e.g. `v1.2.3` since this prevents us from unwanted/accidental updates that also might go wrong in case an update contained breaking changes.